kolibri/content/serializers.py

```diff
@@ -322,15 +321,7 @@ def get_importable(self, obj):
         if 'request' not in self.context or not self.context['request'].query_params.get('importing_from_drive_id', None) or obj.kind == content_kinds.TOPIC:
             return True
         else:
-            # check if the external drive exists given drive id
-            drive_id = self.context['request'].query_params.get('importing_from_drive_id', None)
```

Comment:
If we add the caching logic in here, I think this would work the same, and require less code movement?